variants/STM32F3xx/F303R(6-8)T_F334R(6-8)T/variant_NUCLEO_F334R8.h

```diff
@@ -0,0 +1,192 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+#pragma once
+
+/*----------------------------------------------------------------------------
+ *        STM32 pins number
+ *----------------------------------------------------------------------------*/
+#define PA0                     PIN_A0
```

Comment:
Pins should be ordered to follow Boards connector, mainly the Arduino uno.

Reply:
OK, will look into that. Im guessing you mean defining the layout like 
![image](https://user-images.githubusercontent.com/35117191/187461164-0c184f79-40cc-4503-bef3-0c7cfecd1802.png)
I will give that a shot

Reply:
defined the pins according to the image above, as found in the nucleo64 [user manual](https://www.st.com/resource/en/user_manual/um1724-stm32-nucleo64-boards-mb1136-stmicroelectronics.pdf)